apps/stylelint.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     'selector-pseudo-class-no-unknown': [
       true,
       {
-        ignorePseudoClasses: ['export'],
+        ignorePseudoClasses: ['export', 'global'],
```

Comment:
This is necessary in order to use the `:global()` selector [here](https://github.com/code-dot-org/code-dot-org/pull/58316/files#diff-a078db85eef8d96721b7fc0a110ffffff555194a5a134bede6cefe4e24cf9203R44). Which is necessary to modify styles of a child component in an external library. The alternative is to use `.scss` without modules, which teacher tools is trying to move away from.